[2023-06-05T17:53:22.332+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather.add_data_in_weather manual__2023-06-05T17:34:13.051641+00:00 [queued]>
[2023-06-05T17:53:22.349+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather.add_data_in_weather manual__2023-06-05T17:34:13.051641+00:00 [queued]>
[2023-06-05T17:53:22.349+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-06-05T17:53:22.349+0000] {taskinstance.py:1283} INFO - Starting attempt 12 of 17
[2023-06-05T17:53:22.349+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-06-05T17:53:22.373+0000] {taskinstance.py:1303} INFO - Executing <Task(PostgresOperator): add_data_in_weather> on 2023-06-05 17:34:13.051641+00:00
[2023-06-05T17:53:22.378+0000] {standard_task_runner.py:55} INFO - Started process 12089 to run task
[2023-06-05T17:53:22.382+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather', 'add_data_in_weather', 'manual__2023-06-05T17:34:13.051641+00:00', '--job-id', '669', '--raw', '--subdir', 'DAGS_FOLDER/weather_DAG.py', '--cfg-path', '/tmp/tmp82oumaie']
[2023-06-05T17:53:22.384+0000] {standard_task_runner.py:83} INFO - Job 669: Subtask add_data_in_weather
[2023-06-05T17:53:22.466+0000] {task_command.py:388} INFO - Running <TaskInstance: weather.add_data_in_weather manual__2023-06-05T17:34:13.051641+00:00 [running]> on host 05172cfd5782
[2023-06-05T17:53:22.735+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=fox
AIRFLOW_CTX_DAG_ID=weather
AIRFLOW_CTX_TASK_ID=add_data_in_weather
AIRFLOW_CTX_EXECUTION_DATE=2023-06-05T17:34:13.051641+00:00
AIRFLOW_CTX_TRY_NUMBER=12
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-05T17:34:13.051641+00:00
[2023-06-05T17:53:22.737+0000] {sql.py:254} INFO - Executing: 
            insert into weather_data (date, current_temperature, feels_like, sky, chance_of_precipitation, wind, 
            pressure, uv_index, humidity, precipitation) values 
            ('2023-06-05T19:53:18+02:00',
            '+17째',
            '+19째C',
            'Overcast, no precipitation',
            '40',
            '1',
            '741',
            '1/12',
            '69',
            '0.05'
            )
            
[2023-06-05T17:53:22.753+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-05T17:53:23.159+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-05T17:53:23.164+0000] {sql.py:375} INFO - Running statement: 
            insert into weather_data (date, current_temperature, feels_like, sky, chance_of_precipitation, wind, 
            pressure, uv_index, humidity, precipitation) values 
            ('2023-06-05T19:53:18+02:00',
            '+17째',
            '+19째C',
            'Overcast, no precipitation',
            '40',
            '1',
            '741',
            '1/12',
            '69',
            '0.05'
            )
            , parameters: None
[2023-06-05T17:53:23.165+0000] {sql.py:384} INFO - Rows affected: 1
[2023-06-05T17:53:23.184+0000] {taskinstance.py:1326} INFO - Marking task as SUCCESS. dag_id=weather, task_id=add_data_in_weather, execution_date=20230605T173413, start_date=20230605T175322, end_date=20230605T175323
[2023-06-05T17:53:23.242+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-06-05T17:53:23.267+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
